### ISSUE TITLE
What is the accuracy of the pre-trained model?

### ISSUE THREAD

**Issue by xiyuanssxx** [duanyiqun/DS-DDPM#11]
I want to use it on my own time series data set. In addition, it is difficult for me to find the part of the pre-training model

**Comment by duanyiqun** [comment]
The accuracy of pretraining model on BCI-IV is around 95%, as the classification between different subjects is quite high. 
Yet, I agree that to find a very accurate time series classifier is compared hard. Find a proper guidance is important for performance.

**Comment by xiyuanssxx** [comment]
I’m really happy to see your reply, thx！！！！
